app/components/questions/BelongingRates.js

```diff
@@ -180,12 +208,12 @@ export default function BelongingRates(props) {
         <FailureNotification error={error} data={data} />
       </div>
       <div className="center">
-        <ExamplePieChart data={chartData} />
+        <ExamplePieChart data={chartData} dataFlag={dataFlag} />
         <br />
       </div>
       <div className="center medium-width">
         <p>
-        <span>Based on data from 2018, {areaData.belongingRateFormatted} of people in {areaData.name} agree or strongly agree that they feel a sense of belonging in their community.</span>
+        <GenerateMessage areaData={areaData} dataFlag={dataFlag} />
```

Comment:
Would be nice to have a more descriptive component name, like `AreaBelongingMessage`.